package.json

```diff
@@ -55,15 +55,18 @@
     "eslint-plugin-react-hooks": "^4.5.0",
     "husky": "^8.0.1",
     "istanbul-lib-coverage": "^3.2.0",
+    "lerna": "^5.4.3",
     "lint-staged": "^13.0.3",
+    "nx": "^14.5.7",
     "open-cli": "^7.0.1",
     "prettier": "^2.6.2",
-    "react-router-dom": "^6.3.0",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
     "rollup-plugin-copy": "^3.4.0",
-    "sass": "^1.54.4",
```

Comment:
do we still need this to compile `scss` files?

Reply:
Oh, I removed it

Reply:
oh weird! maybe `vite` includes a `sass` compiler out-of-the-box?